src/lib/express.js

```diff
@@ -207,7 +207,7 @@ function initSwaggerAPI(app) {
 				title: config.app.title,
 				description: config.app.description,
 				contact: {
-					email: _.get(config.mailer, 'from')
+					email: config.mailer?.path
```

Comment:
@jrassa - this looks like a functional change that would break with the current default config. Is this expected, or should we still be using `from` here?